lib/beaker-puppet/install_utils/puppet5.rb

```diff
@@ -0,0 +1,109 @@
+module BeakerPuppet
+  module Install
+    module Puppet5
+
+      # grab build json from the builds server
+      #
+      # @param [String] project_name Name of the project
+      # @param [String] project_sha SHA ref of the build for the project
+      #
+      # @return [String, String] URL to the build for the SHA of the project,
+      #   build json parsed into a ruby hash
+      def fetch_build_details(project_name, project_sha)
+        buildserver_url = 'http://builds.delivery.puppetlabs.net'
+        buildserver_projectsha_url = "#{buildserver_url}/#{project_name}/#{project_sha}"
+        file_folder = "#{buildserver_projectsha_url}/artifacts"
+        filename = "#{project_sha}.yaml"
+        dst_folder = Dir.pwd
```

Comment:
👍  to use `mktmpdir`, 1) because it's a secure directory, and 2) because it will be unique for each run, so any build artifacts from a previous run, e.g. `puppet-agent-latest-x64.msi`, won't pollute future runs.